modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -235,81 +234,85 @@ mysql -u root -h localhost -e "drop user oc_admin@localhost"
 
 # Install owncloud server with the cli
 sudo -u www-data $installation_path/occ maintenance:install \
-
   --database='mysql' --database-name='owncloud' --database-user='root' \
   --database-pass=` --admin-user='admin' --admin-pass='admin'
 ----
 
-=== Running Acceptance Tests for a Suite
-
+=== Types of Acceptance Tests
 
-Run a command like the following:
 
-.. code-block:: bash
+There are 3 types of acceptance tests; API, CLI and webUI.
 
-  sudo -u www-data ./run.sh --suite apiTrashbin
+- API tests test the ownCloud public APIs.
+- CLI tests test the `occ` command-line commands.
+- webUI tests test the browser-based user interface.
 
-=== Running Acceptance Tests for a Feature
+webUI tests require an additional environment to be set up.
+See xref:developer_manual:core/ui-testing.adoc[the UI testing documentation] for more information.
+API and CLI tests are run by using the `test-acceptance-api` and `test-acceptance-cli` make commands.
 
+=== Running Acceptance Tests for a Suite
 
 Run a command like the following:
 
-.. code-block:: bash
+[source,bash]
+----
+sudo -u www-data make test-acceptance-api BEHAT_SUITE=apiTags
+sudo -u www-data make test-acceptancecli BEHAT_SUITE=cliProvisioning
```

Comment:
I have pushed a commit to fix this
s/test-acceptancecli/test-acceptance-cli

GitHub is currently having problems, so sometimes I see the commit and sometimes not.